contrib/mathjax-compat/README.md

```diff
@@ -0,0 +1,15 @@
+# MathJax Compatibility Extension
```

Comment:
That's a good point. I don't actually use MathJax directly, but through a Markdown parser that generates output in using the custom script types that MathJax can understand. I figured that was the normal interface, but reading the docs I see I'm wrong.